lib/src/__tests__/DatePickerTestingLib.test.tsx

```diff
@@ -1,15 +1,43 @@
 import * as React from 'react';
+import deLocale from 'date-fns/locale/de';
+import enLocale from 'date-fns/locale/en-US';
 import TextField from '@material-ui/core/TextField';
 import { getByMuiTest } from './test-utils';
-import { screen, waitFor } from '@testing-library/react';
-import { utilsToUse, FakeTransitionComponent } from './test-utils';
+import { MaterialUiPickersDate } from '../typings';
 import { createClientRender, fireEvent } from './createClientRender';
-import { DatePicker, MobileDatePicker, DesktopDatePicker } from '@material-ui/pickers';
+import { render as rtlRender, screen, waitFor } from '@testing-library/react';
+import { FakeTransitionComponent, DateAdapter, utilsToUse } from './test-utils';
+import {
+  DatePicker,
+  DatePickerProps,
+  DesktopDatePicker,
+  LocalizationProvider,
+  MobileDatePicker,
+} from '@material-ui/pickers';
+
+interface FormProps {
+  locale: any;
+  picker: React.FC<DatePickerProps>;
+  pickerProps: Partial<DatePickerProps>;
+}
+const Form: React.FC<FormProps> = ({ locale = 'en', picker: Picker, pickerProps }) => {
```

Comment:
What about we move this component as close as possible to the next tests? We try to keep each test as isolated as possible. We could have it close to the `const locales = {` variable declaration.